test/Microsoft.ML.Functional.Tests/ModelFiles.cs

```diff
@@ -230,15 +231,17 @@ public class ModelOutput
         [Fact]
         public void LoadModelWithOptionalColumnTransform()
         {
```

Comment:
Does this fix invalidate the old test?

It appears that the the previous test was testing this using SchemaDefinition and to reproduce this issue you are using the one DataViewSchema. 

Does it make sense to test both code paths not replace the previous one?

Reply:
No, the old test is still valid. I'm ok with test both code path not just replace as I don't find a test is test the previous case directly.

This test was originally created at below PR:
[https://github.com/dotnet/machinelearning/pull/3700](https://github.com/dotnet/machinelearning/pull/3700)

From the conversation you can see the original intention was to use the modified way to write test(load model with schema output then use the schema to create prediction engine), but with the type bug this test was written in the way with extra schema definition.

I will made the change to test both ways of create prediction engine, thanks


---
In reply to: [440394844](https://github.com/dotnet/machinelearning/pull/5228#discussion_r440394844) [](ancestors = 440394844)